scripts/cli/src/commands/create_account_identity.ts

```diff
@@ -0,0 +1,45 @@
+
+
+import chalk from 'chalk';
+import { getAPI, generateEntity } from "./util/init";
+import createIdentities from "./helpers/identity_helper";
+//const common = require('./common/common_functions');
+//const input = require('./IO/input');
+
+
+export default async function executeApp (entityName: string) {
+ // common.logAsciiBull();
+  console.log("********************************************");
+  console.log("Welcome to the Command-Line Account Identity Generator.");
+  console.log("********************************************");
+  console.log("The following script will create a new Identity according to the parameters you entered.");
+  
+//   await setup();
+
+  try {
+    // Get API
+    let api = getAPI();
+
+    // Get Entities
+    let entity = await generateEntity(api, entityName);
+    let alice = await generateEntity(api, "Alice");
```

Comment:
Ideally we should default to `Alice` but allow the user to pass in another private key (or mnomic) which can be used as the cdd_provider instead. We can also rename variables to make it clear this is the cdd_provider account (which may happen to be Alice, but may not on a different chain).

Reply:
makes sense, I will add this to the changes i'd be making